index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
#### Information :information_source:
<!DOCTYPE html> is a reference of HTML5 (Last version of Hyper Text Markup Language) This helps the browser to recognize some properties that in
previous versions it wasn’t possible. This link will interest to You: http://anavallasuiza.com/share/presentations/html5/#what-is-html5